pkg/ovsdb/ovsdb.go

```diff
@@ -221,6 +222,35 @@ func (self *OvsDriver) GetOvsPortForContIface(contIface, contNetnsPath string) (
 	return fmt.Sprintf("%v", port["name"]), true, nil
 }
 
+func (self *OvsDriver) FindInterfacesWithError() ([]string, error) {
```

Comment:
Would you please add a functional test where you exercise this new functionality? e.g. after connecting a netns to the OVS bridge, delete one of the created veth pairs. That should render the OVS Interface invalid.

Reply:
Sure :slightly_smiling_face: I've just added the test and force pushed my changes. However, I still need to address your other comments and the DCO signoff.